ci-operator/step-registry/hypershift/aws/run-e2e/external/hypershift-aws-run-e2e-external-commands.sh

```diff
@@ -13,5 +13,30 @@ function cleanup() {
 }
```

Comment:
For posterity, I don't really understand why we need this step since it's a carbon copy of the non-"external" one.